File: SoalB/PHP/DISK.php
<?php

class DISK
{
    /* atribut */
    private $priceDISK, $typeDISK, $capacityDISK;

    /* construct */
    function __construct($priceDISK, $typeDISK, $capacityDISK)
    {
        $this->priceDISK = $priceDISK;
        $this->typeDISK = $typeDISK;
        $this->capacityDISK = $capacityDISK;
    }

    /* Membuat get set method harga DISK */
    function setPriceDISK($priceDISK)
    {
        $this->priceDISK = $priceDISK;
    }
    function getPriceDISK()
    {
        return $this->priceDISK;
    }

    /* Membuat get set method tipe DISK */
    function setTypeDISK($typeDISK)
    {
        $this->typeDISK = $typeDISK;
    }
    function getTypeDISK()
    {
        return $this->typeDISK;
    }

    /* Membuat get set method kapasitas DISK */
    function setCapacityDISK($capacityDISK)
    {
        $this->capacityDISK = $capacityDISK;
    }
    function getCapacityDISK()
    {
        return $this->capacityDISK;
    }

    /* menampilkan hasil data DISK */
    function printDISK()
    {
        echo "DISK Price(IDR)       : " . $this->getPriceDISK() . "<br/>";
        echo "DISK Type             : " . $this->getTypeDISK() . "<br/>";
        echo "DISK Capacity(GB)     : " . $this->getCapacityDISK() . "<br/>";
        echo "******************************" . "<br/>";
    }

    /*destructor */
    function __destruct()
    {
    }
}

File: SoalB/Java/DISK.java
public class DISK {
    /* atribut */
    private int priceDISK;
    private String typeDISK;
    private int capacityDISK;

    /* constructor */
    public DISK(int priceDISK, String typeDISK, int capacityDISK) {
        this.priceDISK = priceDISK;
        this.typeDISK = typeDISK;
        this.capacityDISK = capacityDISK;
    }

    public DISK() {

    }

    /* membuat get set method untuk harga DISK */
    public void setPriceDISK(int priceDISK) {
        this.priceDISK = priceDISK;
    }

    public int getPriceDISK() {
        return priceDISK;
    }

    /* membuat get set method untuk tipe DISK */
    public void setTypeDISK(String typeDISK) {
        this.typeDISK = typeDISK;
    }

    public String getTypeDISK() {
        return typeDISK;
    }

    /* membuat get set method untuk kapasitas DISK */
    public void setCapacityDISK(int capacityDISK) {
        this.capacityDISK = capacityDISK;
    }

    public int getCapacityDISK() {
        return capacityDISK;
    }

    /* menampilkan data DISK */
    public void printDISK() {
        System.out.println("DISK Price(IDR)     : " + this.getPriceDISK());
        System.out.println("DISK Type           : " + this.getTypeDISK());
        System.out.println("DISK Capacity(GB)   : " + this.getCapacityDISK());
        System.out.println("######################################");
    }
}
